<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="3" total="7" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-05-29T11:19:36 GMT-07:00" name="Suite" finished-at="2024-05-29T11:20:41 GMT-07:00" duration-ms="65175">
    <groups>
    </groups>
    <test started-at="2024-05-29T11:19:36 GMT-07:00" name="Test" finished-at="2024-05-29T11:20:41 GMT-07:00" duration-ms="65175">
      <class name="testcases.RegisterTestCases">
        <test-method is-config="true" signature="ExtentReport()[pri:0, instance:testcases.RegisterTestCases@6379eb]" started-at="2024-05-29T11:19:35 GMT-07:00" name="ExtentReport" finished-at="2024-05-29T11:19:36 GMT-07:00" duration-ms="969" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentReport -->
        <test-method is-config="true" signature="browserAndurlLaunch()[pri:0, instance:testcases.RegisterTestCases@6379eb]" started-at="2024-05-29T11:19:36 GMT-07:00" name="browserAndurlLaunch" finished-at="2024-05-29T11:20:03 GMT-07:00" duration-ms="26744" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- browserAndurlLaunch -->
        <test-method signature="VerifyRegistrationWithValidData()[pri:1, instance:testcases.RegisterTestCases@6379eb]" started-at="2024-05-29T11:20:03 GMT-07:00" name="VerifyRegistrationWithValidData" finished-at="2024-05-29T11:20:07 GMT-07:00" duration-ms="3991" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyRegistrationWithValidData -->
        <test-method is-config="true" signature="browserAndurlLaunch()[pri:0, instance:testcases.RegisterTestCases@6379eb]" started-at="2024-05-29T11:20:07 GMT-07:00" name="browserAndurlLaunch" finished-at="2024-05-29T11:20:14 GMT-07:00" duration-ms="7175" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- browserAndurlLaunch -->
        <test-method signature="VerifyRegistrationWithBlankData()[pri:2, instance:testcases.RegisterTestCases@6379eb]" started-at="2024-05-29T11:20:14 GMT-07:00" name="VerifyRegistrationWithBlankData" finished-at="2024-05-29T11:20:17 GMT-07:00" duration-ms="3009" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyRegistrationWithBlankData -->
        <test-method is-config="true" signature="endReport()[pri:0, instance:testcases.RegisterTestCases@6379eb]" started-at="2024-05-29T11:20:41 GMT-07:00" name="endReport" finished-at="2024-05-29T11:20:43 GMT-07:00" duration-ms="1639" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
      </class> <!-- testcases.RegisterTestCases -->
      <class name="testcases.LoginTestCases">
        <test-method is-config="true" signature="browserAndurlLaunch()[pri:0, instance:testcases.LoginTestCases@5b239d7d]" started-at="2024-05-29T11:20:17 GMT-07:00" name="browserAndurlLaunch" finished-at="2024-05-29T11:20:25 GMT-07:00" duration-ms="8173" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- browserAndurlLaunch -->
        <test-method signature="VerifyLoginWithInvlidData()[pri:0, instance:testcases.LoginTestCases@5b239d7d]" started-at="2024-05-29T11:20:25 GMT-07:00" name="VerifyLoginWithInvlidData" finished-at="2024-05-29T11:20:28 GMT-07:00" duration-ms="3245" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [Warning: No match for E-Mail Address and/or Password.] but found [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Warning: No match for E-Mail Address and/or Password.] but found [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at resources.commonMethods.handleAssertion(commonMethods.java:17)
at testcases.LoginTestCases.VerifyLoginWithInvlidData(LoginTestCases.java:59)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyLoginWithInvlidData -->
        <test-method is-config="true" signature="browserAndurlLaunch()[pri:0, instance:testcases.LoginTestCases@5b239d7d]" started-at="2024-05-29T11:20:29 GMT-07:00" name="browserAndurlLaunch" finished-at="2024-05-29T11:20:38 GMT-07:00" duration-ms="9480" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- browserAndurlLaunch -->
        <test-method signature="VerifyLoginWithValidData()[pri:0, instance:testcases.LoginTestCases@5b239d7d]" started-at="2024-05-29T11:20:38 GMT-07:00" name="VerifyLoginWithValidData" finished-at="2024-05-29T11:20:41 GMT-07:00" duration-ms="2987" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyLoginWithValidData -->
      </class> <!-- testcases.LoginTestCases -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
